src/freenas/usr/local/lib/middlewared_truenas/plugins/failover_/event_linux.py

```diff
@@ -157,9 +150,23 @@ def event(self, ifname, event):
             if refresh:
                 self.run_call('failover.status_refresh')
 
-    def _zpool_export_sig_alarm(self, sig, tb):
+    def _export_zpools(self, volumes):
 
-        raise ZpoolExportTimeout()
+        # export the zpool(s)
+        try:
+            for vol in volumes:
+                if vol['status'] != 'OFFLINE':
+                    self.middleware.call_sync('zfs.pool.export', vol, {'force': True})
+                    logger.info('Exported "%s"', vol)
+        except Exception as e:
+            # catch any exception that could be raised
+            # We sleep for 5 seconds here because this is
+            # in its own thread. The calling thread waits
+            # for self.ZPOOL_EXPORT_TIMEOUT and if this
+            # thread is_alive(), then we violently reboot
+            # the node
+            logger.error('Error exporting "{}" with error {}'.format(vol, e))
+            time.sleep(5)
```

Comment:
Should we sleep ZPOOL_EXPORT_TIMEOUT + 1?

Reply:
Absolutely should, I'll fix.